pylint/utils.py

```diff
@@ -693,6 +693,16 @@ def register_messages(self, checker):
             self._messages[msg.symbol] = msg
             self._alternative_names[msg.msgid] = msg
             for old_id, old_symbol in msg.old_names:
+                assert (old_id not in self._alternative_names or
+                        self._alternative_names[old_id] == msg), \
+                        'Old message id %r is already defined' % old_id
+                assert (old_symbol not in self._messages or
+                        self._messages[old_symbol] == msg), \
+                        'Old message symbol %r is already defined ' \
+                        'as a current symbol' % old_symbol
+                assert (old_symbol not in self._alternative_names or
+                        self._alternative_names[old_symbol] == msg), \
+                        'Old message symbol %r is already defined' % old_symbol
```

Comment:
Oh, I was referring in general, not necessarily to do it in this PR. If you still do it though, make sure they are different commits.